internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -157,7 +162,7 @@ func reconcileClusterRoleBindingFuncs(clusterRoleBinding rbac.ClusterRoleBinding
 
 func reconcileConfigMapFuncs(configMap core.ConfigMap) common.ReconcileFunc {
 	return func(request *common.Request) (common.ReconcileResult, error) {
-		configMap.Namespace = kubeVirtNamespace
+		configMap.Namespace = getVmConsoleProxyNamespace(request)
 		return common.CreateOrUpdate(request).
 			ClusterResource(&configMap).
 			WithAppLabels(operandName, operandComponent).
```

Comment:
Similarly here, can you add `UpdateFunc()`?

Reply:
Done.